docs/GETTING_STARTED.md

```diff
@@ -29,6 +29,8 @@ https://github.com/awslabs/soci-snapshotter.git`
 Often fixed with `sudo yum install zlib-devel`
 - having the proper go version installed, as of this document please use 1.18+
 
+**fuse** - confirm FUSE is installed for mounting userspace filesystems with `fusermount -V` (optional)
```

Comment:
This change raises more questions than it answers. "Optional prerequisite" sounds like a bit of an oxymoron. A prerequisite--that is, all the items in this section--should be required to use SOCI and/or get through the rest of this Getting Started guide. If we're going to list this as optional, we should describe *why* it's optional; i.e., what won't work as a result of not having it.

Reply:
"optional prerequisite" doesn't sound any more oxymoronic to me than "optional dependency", which is what this is.

re what won't work as a result of not having it, I think the answer to that is fuse mounts without root access. I could add something about that here?

Reply:
> noun: pre-requisite  
>    a thing that is required as a prior condition for something else to happen or exist.

> I could add something about that here

Sounds good.